packages/serverless-contracts/package.json

```diff
@@ -40,6 +40,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.18.3",
+    "cross-fetch": "^3.1.5",
```

Comment:
We can move this to devDependencies I think since it only used for typing

Reply:
By referencing browser fetch module, i was able to remove cross-fetch deps entirely